tests/ui/test_search.py

```diff
@@ -127,6 +120,7 @@ def test_long_terms_dont_break_suggestions(base_url, selenium):
 
 
 @pytest.mark.nondestructive
+@pytest.mark.skip(reason='Search Behavior has changed')
```

Comment:
Let's create an issue if that leads to skipping the test.

Reply:
Trying this again with proper recommendations seems to work fine, so I'll "unskip" it.